text/000x-assethub.md

```diff
@@ -0,0 +1,92 @@
+# RFC-0000: Lowering Deposit Requirements on Polkadot Asset Hub
+
+|                 |                                                                                                    |
+| --------------- | -------------------------------------------------------------------------------------------------- |
+| **Start Date**  | 2 November 2023                                                                                    |
+| **Description** | A proposal to reduce the minimum deposit required for collection creation on the Polkadot Asset Hub, making it more accessible and affordable for artists. |
+| **Authors**     | [Aurora Poppyseed](https://github.com/poppyseedDev), [Just_Luuuu](https://github.com/justLuuuu)    |
```

Comment:
Further down you mention `KodaDot` a few times. I think it should be mentioned here as well - if it is part of a business effort.

Reply:
This initiative stemed from an independent effort, seeing how frustrated Luuu was with converting artists to Asset Hub, however I additionally added VikiiVal as a contributor since he is helping in gathering the requested data.

Reply:
Fixed in https://github.com/poppyseedDev/RFCs/commit/0e49a4fd5dc3cbe4d4e326a6ede38c91f000da10